src/untangled/client/impl/data_fetch.cljs

```diff
@@ -330,9 +331,9 @@
                         (swap! app-state assoc :untangled/server-error error)
                         (doseq [item loading-items]
                           (swap! app-state (fn [s]
-                                             (-> s
-                                                 (update :untangled/loads-in-progress disj (data-uuid item))
-                                                 (update-in (data-path item) set-failed! error))))))
+                                             (cond-> s
+                                               (data-marker? item) (update-in (data-path item) set-failed! error)
+                                               (update :untangled/loads-in-progress disj (data-uuid item)))))))
```

Comment:
`cond->`, not `cond`! We still want to remove from loads-in-progress if `:marker` is false, no?